android/java/org/chromium/chrome/browser/firstrun/WelcomeOnboardingActivity.java

```diff
@@ -211,9 +227,12 @@ private void onClickViews() {
         if (mBtnPositive != null) {
             mBtnPositive.setOnClickListener(
                     view -> {
-                        if (mCurrentStep == 1 && !isDefaultBrowser()) {
+                        // If this is the first step and Brave is not set as default browser
```

Comment:
I would personally remove this comment because it says the same thing the `if` condition does. But feel free to ignore.